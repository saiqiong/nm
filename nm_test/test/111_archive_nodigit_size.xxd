213c 6172 6368 3e0a 746f 746f 2020 2020
2020 2020 2020 2020 3135 3230 3431 3934
3936 2020 3130 3935 3520 3020 2020 2020
3130 3036 3434 2020 6120 2020 2020 2020
2020 600a 7361 6c75 740a               
