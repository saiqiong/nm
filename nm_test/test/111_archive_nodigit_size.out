!<arch>
toto            1520419496  10955 0     100644  a         `
salut
